### DISCUSSION TITLE
Idea processing ide easyer to use

### DISCUSSION THREAD

**Discussion by KisonHe** [Bodmer/TFT_eSPI#1059] [New processor/display support or capability request] [answered]
First of all, it is really excited to see this repo! Thanks you all for making this beautiful lib!
I had seen another tool that can make Anti-aliased Fonts, see this repo 
https://github.com/t123yh/mcufont-encoder
The author is quite busy so he didn't have time to write readme… Basically if you want to use it, you need to write a config.yaml like this one https://gist.github.com/KisonHe/f2036ba927571cc5e4cba53c6484c226
The template is something like this one
https://gist.github.com/KisonHe/17366a426c2c25f84f6f3389794729aa
With a simple command added to Cmakelists, it can auto generate a header file containing all the strings anti-aliasd. If you want to access the string, just include the header and use strings::YourStringName

OK, I am new to online community and not really good at expressing my ideas, sorry for saying things so long. 
The processing ide sketch is a little complicated for specifying unicodeBlocks, it would be more convenient if these blocks can be automatically selected right? I am new to programming world and really don't know how to make this come true, any ideas? Thanks all:D

**Comment by Bodmer** [comment, marked as answer]
Would be worth looking at the processing sketch again and reading the comments, adding a unicode block [just involves removing //](https://github.com/Bodmer/TFT_eSPI/blob/master/Tools/Create_Smooth_Font/Create_font/Create_font.pde#L149) from a line.

The Processing IDE also has a "Create Font..." in the Tools menu to create a font with a given size/character range.
